config/kernel/linux-sunxi64-current.config

```diff
@@ -3926,7 +3927,7 @@ CONFIG_MFD_SUN4I_GPADC=y
 # CONFIG_MFD_BCM590XX is not set
 CONFIG_MFD_BD9571MWV=m
 CONFIG_MFD_AC100=y
-# CONFIG_MFD_AC200 is not set
+CONFIG_MFD_AC200=y
```

Comment:
This needs to be verified on at least Orangepi 3 / LTS.

Reply:
Are you referring to this board [orangepi3.csc](https://github.com/armbian/build/blob/main/config/boards/orangepi3.csc) ？
After modifying the `.csc` suffix to `.conf`, I successfully built the `Orangepi3_bullseye_current_6.1.29.img` without any error messages. 
Do I need to write it into the Orangepi3 hardware board for testing?

Reply:
> Do I need to write it into the Orangepi3 hardware board for testing?

Yes. Actually we need to test all boards that are using this chip - i'll run it our what we have in the test rig. Just to make sure nothing breaks. We are low with time and next build should be release. We are at bug fix changes only.